src/controllers/update/remove.need.to.make.changes.to.entity.controller.ts

```diff
@@ -0,0 +1,30 @@
+import { NextFunction, Request, Response } from "express";
```

Comment:
Given my comments below and the fact that this is basically just an update screen, I wonder if the new controller for remove (and HTML template page) is required.